bitvector1024_test.go

```diff
@@ -0,0 +1,463 @@
+package bitfield
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+)
+
+func TestBitvector1024_Len(t *testing.T) {
+	bv := NewBitvector1024()
+	if bv.Len() != 1024 {
+		t.Errorf("(%x).Len() = %d, wanted %d", bv, bv.Len(), 64)
+	}
+}
+
+func TestBitvector1024_BitAt(t *testing.T) {
```

Comment:
https://github.com/prysmaticlabs/go-bitfield/blob/master/bitvector64_test.go includes an out of bounds case. Is there such a case here? If yes, can you please tag it with a comment?

Reply:
It has one. I added a comment for clarification

https://github.com/prysmaticlabs/go-bitfield/pull/33/commits/47ed996352ff1b924c11b9953fe8b51c5b09afb5#diff-28964f5e352b039b26d03a3a9d92fe63423cee94d9f8d341c63926bb6531d10dR92